File: python/7 kyu/270.py
# https://www.codewars.com/kata/59706036f6e5d1e22d000016/train/python

# Passed

from string import ascii_lowercase

def words_to_marks(s):
    letter_to_score_dict = {
        letter: pos for pos, letter in enumerate(ascii_lowercase, start=1)
    }
    
    score = sum([letter_to_score_dict[letter] for letter in s.lower()])
    return score

output = words_to_marks('attitude')
print(output)

File: nodejs/7 kyu/270.js
// https://www.codewars.com/kata/5546180ca783b6d2d5000062/train/javascript

// Passed

function squares(x, n) {
  let result = [];
  for (let i = 0; i < n; i++) {
    let elem = x ** (2 ** i);
    result.push(elem);
  }

  return result;
}

const output = squares(2, 5);
console.log(output);